src/FixedDataTableBufferedRows.js

```diff
@@ -129,6 +129,9 @@ class FixedDataTableBufferedRows extends React.Component {
       if (rowIndex === undefined) {
         rowIndex =
           this._staticRowArray[i] && this._staticRowArray[i].props.index;
+        if (rowIndex === undefined) {
```

Comment:
Earlier, if `rowIndex` was undefined, then we set the prop `fake` to true inside the `renderRow`.
The `fake` prop is only used to distinguish rows without an index, and it's only used by <FixedDataTableRow> to render null.

Instead of doing all this, we could simply render null here in `FixedDataTableBufferedRows`.
Rows aren't expected to be rendered without an index anyway.